src/main/resources/project/project.xml

```diff
@@ -6,6 +6,83 @@
     <description>Integration with WebSphere</description>
     <workspaceName />
     <propertySheet>
+        <!-- CEV-17558 improvements -->
+        <property>
+            <propertyName>ec_config</propertyName>
+            <description/>
+            <propertySheet>
+                <!-- WebSphere doesn't have custom GWT logic. Setting it for plugin context -->
+                <property>
+                    <propertyName>ec_formXmlCompliant</propertyName>
+                    <value>true</value>
+                </property>
+                <property>
+                    <propertyName>configLocation</propertyName>
+                    <expandable>0</expandable>
+                    <value>websphere_cfgs</value>
+                </property>
+                <property>
+                    <propertyName>fields</propertyName>
```

Comment:
Lets keep the list of fields to display in the list page limited to 2-3 including the configuration name which is the default.
So, for websphere, just add the websphere url here. This is also consistent with the fields we are currently showing for the plugin in the plugin manager page.